src/app/(sidebar)/explore-endpoints/[[...pages]]/page.tsx

```diff
@@ -175,33 +231,42 @@ export default function ExploreEndpoints() {
   }, [isSuccess, isError]);
 
   const buildUrl = useCallback(() => {
-    const mapPathParamToValue = (pathParams: string[]) => {
-      return pathParams.map((pp) => params[pp] ?? pp).join("/");
-    };
+    const parseUrlPath = (path: string) => {
+      const pathArr: string[] = [];
+
+      path.split("/").forEach((p) => {
+        const param = p.match(REGEX_TEMPLATE_PATH_PARAM_VALUE)?.[0];
+
+        if (param) {
+          return pathArr.push(params[param] ?? "");
+        }
 
-    const endpointPath = `/accounts${pageData?.endpointPathParams ? `/${mapPathParamToValue(pageData.endpointPathParams.split(","))}` : ""}`;
-    const endpointParams = pageData?.endpointParams;
+        return pathArr.push(p);
+      });
 
-    const baseUrl = `${endpointNetwork.horizonUrl}${endpointPath}`;
+      return pathArr.join("/");
```

Comment:
<img width="501" alt="double-slash" src="https://github.com/stellar/laboratory/assets/3912060/6d5f6848-34aa-4f09-9ca7-3b76086aa83f">
it creates double slashes, `/effects` as an example because of an empty string in between.
`['', 'liquidity_pools', '', 'effects']`